aws/data_source_aws_ec2_managed_prefix_list.go

```diff
@@ -0,0 +1,153 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func dataSourceAwsEc2ManagedPrefixList() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceAwsEc2ManagedPrefixListRead,
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
Nit: It would be easier to scan these attributes if they were sorted alphabetically and we may eventually enable [`tfproviderlint` XS002](https://github.com/bflad/tfproviderlint/tree/master/xpasses/XS002)